src/Components/ContactsListComponents/ContactsList.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
Please rename component folder from ContactsListComponent to ContactList.
and rename the file name from ContactList to index.js.

this makes the import statement more readable.